comms/src/connectivity/manager.rs

```diff
@@ -271,8 +273,20 @@ impl ConnectivityManagerActor {
                 let _ = reply.send(states);
             },
             BanPeer(node_id, duration, reason) => {
-                if let Err(err) = self.ban_peer(&node_id, duration, reason).await {
-                    error!(target: LOG_TARGET, "Error when banning peer: {:?}", err);
+                if !self.allow_list.contains(&node_id) {
```

Comment:
Just add a message here that this Ban Request was ignored due to the AllowList so that this isn't ignored silently.

Reply:
Will do.